lib/Display/DisplayTemplateDriver.cpp

```diff
@@ -517,13 +510,21 @@ void DisplayTemplateDriver::renderLines(JsonArray lines) {
 
 std::shared_ptr<Region> DisplayTemplateDriver::addBitmapRegion(uint16_t x,
     uint16_t y,
-    uint16_t w,
-    uint16_t h,
     uint16_t color,
     uint16_t backgroundColor,
     VariableFormatterFactory& formatterFactory,
     JsonObject spec,
     uint16_t index) {
+    uint16_t w, h;
+    Serial.println(__LINE__);
+    Serial.println(__LINE__);
+    //String bitmap = formatterFactory.create(spec)->format(spec["variable"].as<const char*>());
+    String variable = spec["variable"].as<const char*>();
+    //
+    Serial.println();
+    BitmapRegion::getBitmapDimensions(vars.get(variable), &w, &h);
+    Serial.println(__LINE__);
+    Serial.println(__LINE__);
```

Comment:
Don't think we need (or should) get the initial width/height.  We can just pass 0 into the parent class for both width and height (this is what `TextRegion` does, which also has undefined dimensions initially).

Reply:
For a while it refused to show it on boot unless I did that first. It's possible it's changed now. Will test and report back.

Reply:
Seems fine without it. Will keep it in mind in case it breaks again,